js/githubToken.js

```diff
@@ -0,0 +1,5 @@
+(function(module){
+  var token = {};
+  token.githubToken = '2775fc034c4ff7be7bca5bda33437199d71aa873';
```

Comment:
eek! regenerate this please. always store githubToken in a file declared in .gitignore for sensitive tokens